### PR TITLE
feat: [CDS-54519]:ssh-editing-credentials-fix

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Fixed issue related editing ssh on org and account level
Fixed issue related empty config file

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger cypress-pipeline

File: src/modules/30-secrets/modals/CreateSSHCredModal/CreateSSHCredWizard.tsx [hunk 1]
```diff
@@ -23,15 +23,20 @@ interface CreateSSHCredWizardProps {
   hideModal?: () => void
 }
 
+export interface SecretSSHParams {
+  orgIdentifier?: string
+  projectIdentifier?: string
+}
```
Comment (rameshx) [inline src/modules/30-secrets/modals/CreateSSHCredModal/CreateSSHCredWizard.tsx:29]: Can we use an existing type/interface instead of defining a new one?

**Review by rameshx — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/30-secrets/modals/CreateSSHCredModal/CreateSSHCredWizard.tsx (hunk 1 above)
Comment (sunnykesh-harness) [reply src/modules/30-secrets/modals/CreateSSHCredModal/CreateSSHCredWizard.tsx:29]: Please check `ResourceScope` Interface defined in 20-rbac. We can use as it is or Partail<ResourceScope>

**Review by sunnykesh-harness — APPROVED** [review]

**Comment by litvinovstas** [comment]
retrigger cypress-pipeline

**Comment by vkbansal-harness** [comment]
Fixes for the failing tests are merged to develop

**Pull request merged**